### PR TITLE
draft blog post contributing guidelines

### PR DESCRIPTION
**Pull request by shannonpileggi**

### CODE REVIEW

**Comment by shannonpileggi** [comment]
@yabellini i ammeded the workflow instructions to say use whatever method you are comfortable with etc, instead of writing out another set of instructions. we can always add later if it is a common ask, but i think this is sufficient for now. do you agree?

i also ammended the wording with regards to knitting vs site rendering.

**Comment by yabellini** [comment]
Looks good to me. Thanks so much!

**Comment by shannonpileggi** [comment]
thanks @yabellini! we need at least one reviewer to approve before we merge.

**Review by yabellini — APPROVED** [review]
Thanks so much for writing this guidelines!

File: content/comm/blog/index.en.md [hunk 1]
```diff
@@ -4,4 +4,208 @@ menuTitle: "Blog"
...
\ No newline at end of file
+## Editors
+
+Currently members of the Leadership team:
+
+- Shannon Pileggi [`@shannonpileggi`](https://github.com/shannonpileggi)
+
+- Yanina Bellini Saibene [`@yabellini`](https://github.com/yabellini)
+
+- Athanasia M. Mowinckel [`@ChrisZasa`](https://github.com/Athanasiamoa)
```
Comment (drmowinckels) [inline content/comm/blog/index.en.md:15]:
```suggestion
- Athanasia M. Mowinckel [`@drmowinckels`](https://github.com/drmowinckels)
```

File: content/comm/blog/index.en.md [hunk 2]
```diff
@@ -4,4 +4,208 @@ menuTitle: "Blog"
...
+
+## Website maintainers
+
+- Athanasia M. Mowinckel [`@Athanasiamo`](https://github.com/Athanasiamo)
```
Comment (drmowinckels) [inline content/comm/blog/index.en.md:19]:
```suggestion
- Athanasia M. Mowinckel [`@drmowinckels`](https://github.com/drmowinckels)
```

File: content/comm/blog/index.en.md [hunk 3]
```diff
@@ -4,4 +4,208 @@ menuTitle: "Blog"
...
+---
+title: "1. Behind the scenes of R-Ladies IWD2018 Twitter action!"
+author: "R-Ladies"
+date: "2018-03-26"
+description: "Part 1: Ideation and Creation!"
+tags:
+- iwd
+- twitter
+- part1
+- 2018
```
Comment (drmowinckels) [inline content/comm/blog/index.en.md:131]:
```suggestion
```

File: content/comm/blog/index.en.md [hunk 4]
```diff
@@ -4,4 +4,208 @@ menuTitle: "Blog"
...
+- a new folder is created like `yyyy-mm-dd-your-post-title`
+
+- with an outline for your post named `index.Rmd`
+
+Add all files needed for the post (images, data files, gifs, etc.) should be 
+added to the new folder `content/post/yyyy-mm-dd-your-post-title`. 
+
+As you write you post, remember to `knit` your post to see how it looks.
+The blogdown site will use the markdown file created from your knitted `index.Rmd` 
+on the site, not your `Rmd` itself.
```
Comment (drmowinckels) [inline content/comm/blog/index.en.md:151]:
```suggestion
on the site, not your `Rmd` itself.  If your post does not include any code chunks 
(i.e. no code is being run), you can write a single `.md` file instead, and not need
to knit at all.
```

**Review by drmowinckels — APPROVED** [review]
Fixes my user links a couple places, and a couple of extra suggestions.

**Comment by shannonpileggi** [comment]
@drmowinckels i think this is good to merge once checks pass

**Pull request merged**